### PR TITLE
Json content update

### PR DESCRIPTION
**Pull request by B3nz01d**
Updates of the json file with new content and categories

### CODE REVIEW

File: demo/src/main/res/raw/recipes.json [hunk 1]
```diff
@@ -247,12 +254,13 @@
           "quantity": "30ml"
         }
       ],
-      "description": "",
+      "description": "A wonderfull pasta dish that smells like a piece of Italy.",
```
Comment (julien-deramond) [inline demo/src/main/res/raw/recipes.json:257]:
```suggestion
      "description": "A wonderful pasta dish that smells like a piece of Italy.",

```

File: demo/src/main/res/raw/recipes.json [hunk 2]
```diff
@@ -272,207 +280,837 @@
           "quantity": "5ml"
         }
       ],
-      "description": "",
+      "description": "A simple revisit of the pizza that makes your ordinary a little mor extraordinary.",
```
Comment (julien-deramond) [inline demo/src/main/res/raw/recipes.json:283]:
```suggestion
      "description": "A simple revisit of the pizza that makes your ordinary a little more extraordinary.",

```

File: demo/src/main/res/raw/recipes.json [hunk 3]
```diff
@@ -272,207 +280,837 @@
...
+        {
+          "foodId": 41,
+          "quantity": "12 cl"
+        },
+        {
+          "foodId": 45,
+          "quantity": "fill up the glass"
+        }
+      ],
+      "description": "The mojito is one of the world's most popular cocktails, It is originating from cuba, and this virgin version is the perfect summer drink for the entire family.",
```
Comment (julien-deramond) [inline demo/src/main/res/raw/recipes.json:333]:
```suggestion
      "description": "The mojito is one of the world's most popular cocktails, It is originating from Cuba, and this virgin version is the perfect summer drink for the entire family.",

```

File: demo/src/main/res/raw/recipes.json [hunk 4]
```diff
@@ -272,207 +280,837 @@
...
+        {
+          "foodId": 53,
+          "quantity": "8 leaves"
+        },
+        {
+          "foodId": 13,
+          "quantity": "a limp"
+        }
+      ],
+      "description": "A basic marvel from the italian cuisine, very simple and quick for the summer.",
```
Comment (julien-deramond) [inline demo/src/main/res/raw/recipes.json:433]:
```suggestion
      "description": "A basic marvel from the Italian cuisine, very simple and quick for the summer.",

```

**Comment by julien-deramond** [comment]
Sorry I received the notification for this PR. Noticed a few typos and made some suggestions. I haven't checked the entire file. I'll let you finish the review :)

**Review by paulinea — DISMISSED** [review]

**Review by paulinea — APPROVED** [review]

**Pull request merged**